api/catalog/api/serializers/media_serializers.py

```diff
@@ -42,6 +48,8 @@ class MediaSearchRequestSerializer(serializers.Serializer):
         "extension",
         "mature",
         "qa",
+        # "unstable__sort_by",  # excluding unstable fields
+        # "unstable__sort_dir",  # excluding unstable fields
```

Comment:
Yes, that's correct. I added them so that when someone is searching for references to update (when these fields are being promoted to stable), these two lines also show up in the search.

Reply:
Perfect :) That's a really thoughtful addition.